### DISCUSSION TITLE
Is there a single swanctl command that could start multiple tunnels?

### DISCUSSION THREAD

**Discussion by atomus1990** [strongswan/strongswan#1513] [Q&A] [answered]
I have a configuration with multiple tunnels. I can start each tunnel using:
 "swanctl --initiate --child net-net"
 "swanctl --initiate --child net1-net1"
I would like to know if there is a way to start multiple tunnels with a single command.
In the older implementation, is possible to run "ipsec start".

**Comment by tobiasbrunner** [comment, marked as answer]
> I would like to know if there is a way to start multiple tunnels with a single command.
> In the older implementation, is possible to run "ipsec start".

Not with a `swanctl` command (I guess you could write a script or a separate vici client that automates it), but since you mention `ipsec start`, which does not start any tunnels per se, maybe `start_action` is what you are looking for? Perhaps also read [this](https://docs.strongswan.org/docs/5.9/howtos/introduction.html#_invocation_and_maintenance).

**Reply by atomus1990** [reply]
Thank you @tobiasbrunner, you answered my question :)
